### DISCUSSION TITLE
Windows version is weak

### DISCUSSION THREAD

**Discussion by kleenkanteen** [chidiwilliams/buzz#497] [General]
Why is the windows version so feature poor? I can't hit a shortcut and type with my voice anywhere I want. I am limited.

And what made this program cross platform? Is the mac version coded in swift?

**Comment by faveoled** [comment]
> And what made this program cross platform? Is the mac version coded in swift?

Python and Qt

**Reply by kleenkanteen** [reply]
If it is using Qt Then why does the windows version look different and have different features

**Reply by faveoled** [reply]
I guess it uses a Windows theme. No idea about the shortcuts though

**Reply by kleenkanteen** [reply]
@chidiwilliams any ideas

**Reply by chidiwilliams** [reply]
Hey, so I built this Buzz version cross-platform with Qt first. Then I faced some problems with the Mac version and rebuilt the app with Swift (the version on the App Store). SwiftUI was a lot easier to work with, so it was easy to add lots of more features to the app that are difficult with Qt.

> I can't hit a shortcut and type with my voice anywhere I want

This is an interesting feature request in any case, and I'll take a look at it soon and see if/how I can implement it here. Thanks.
